tests/pages/history.py

```diff
@@ -0,0 +1,35 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+
+from base import BasePage
+
+
+class HistoryPage(BasePage):
+
+    _url = '{base_url}/{locale}/about/history'
+
+    _slideshow_locator = (By.CSS_SELECTOR, '#slideshow-stage > .on')
+    _previous_button_locator = (By.CSS_SELECTOR, '.slide-control > .prev')
+    _next_button_locator = (By.CSS_SELECTOR, '.slide-control > .next')
+    _list_locator = (By.CSS_SELECTOR, '#slideshow > .slides')
+
+    @property
+    def is_slideshow_displayed(self):
+        self.wait.until(expected.presence_of_element_located(self._slideshow_locator))
```

Comment:
Hmm, no it's not. The `on` class is only added when the slideshow is displayed. I'll give this a rethink and refactor.